app/decorators/manageiq/providers/kubernetes/container_manager_decorator.rb

```diff
@@ -0,0 +1,7 @@
+class ManageIQ::Providers::Kubernetes::ContainerManagerDecorator < Draper::Decorator
+  delegate_all
+
+  def fonticon
+    @fonticon ||= "pficon-kubernetes".freeze
```

Comment:
I guess not, the original version of this PR put those in the model .. and that's what all the models do for `ems_type` and `description`, so I figured there must be a reason